.gitlab-ci.yml

```diff
@@ -281,43 +282,56 @@ topPRMEMC-sim:
   variables:
     VIVADO_VERSION: "2019.2"
     PROJ_NAME: "PRMEMC"
-  needs: # needed to avoid interference from ME-vitis-hls-build
+  needs:
+    - download
+    - PR-quality-check
+    - ME-quality-check
+    - MC-quality-check
+topTETC-sim:
+  <<: *template_topTF-sim
+  variables:
+    VIVADO_VERSION: "2019.2"
+    PROJ_NAME: "TETC"
+  needs:
     - download
-    - PR-vivado-hls-build
-    - ME-vivado-hls-build
-    - MC-vivado-hls-build
+    - TE-quality-check
+    - TC-quality-check
```

Comment:
Same comment about wanting the build jobs to complete before running the simulation of the chain.

Reply:
Same reply as above.